### PR TITLE
Create or Retrieve profile after login

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues:

* [AEROGEAR-7878](https://issues.jboss.org/browse/AEROGEAR-7878)

### CODE REVIEW

**Review by ziccardi — APPROVED** [review]
LGTM

File: Memeolist/Memeolist/AppDelegate.swift [hunk 1]
```diff
@@ -39,4 +44,38 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
         return false
     }
+    
+    public func createOrRetrieveProfile(user: User) {
```
Comment (wtrocki) [inline Memeolist/Memeolist/AppDelegate.swift]: [Optional] Move this out of the Application as it's specific business logic.

**Comment by wtrocki** [comment]
When trying to open application now to test this application crashes:
screen shot 2018-09-13 at 9 49 07 am

It seems unrelated to this change, but prevent's me from testing this carefully. 
I have tried master and problem is not happening there.

**Comment by ziccardi** [comment]
@wtrocki : that's weird. I tested it many times yesterday, even together with @danielpassos and it worked.

**Comment by wtrocki** [comment]
```
        "realm": "Memeolist",
        "resource": "ios-app",
        "ssl-required": "external",
        "public-client": true
```
Using this

**Comment by danielpassos** [comment]
@wtrocki I have no idea how to reproduce that.

Step by step

**Comment by wtrocki** [comment]
@danielpassos This seems to be Auth issue related to my emulator. 
I switched to phone and things started to work fine. Good to merge!

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Ok. So this problem can happen and we going to resolve that in the @wojta  PR

**Pull request merged**